### PR TITLE
Add animationDelay prop to Coachmark

### PR DESCRIPTION
**Pull request by leddie24**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

- Changes default animation delay for Coachmark to 0s.
- Add `delayBeforeCoachmarkAnimation` prop for Coachmark

#### Focus areas to test

(optional)

### CODE REVIEW

File: common/changes/@uifabric/styling/edwl-2018-10-fixCoachmarkDelay_2018-10-03-21-59.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/styling",
+      "comment": "add delayLength parameter to _createDefaultAnimation",
```
Comment (Vitalius1) [inline common/changes/@uifabric/styling/edwl-2018-10-fixCoachmarkDelay_2018-10-03-21-59.json]: Nit: a little unclear where this `_createDefaultAnimation` in the styling package is located.

**Comment by Vitalius1** [comment]
@leddie24 Looks like you need a snapshot update.

File: common/changes/@uifabric/styling/edwl-2018-10-fixCoachmarkDelay_2018-10-03-21-59.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/styling",
+      "comment": "add delayLength parameter to _createDefaultAnimation in PulsingBeaconAnimationStyles",
+      "type": "patch"
```
Comment (leddie24) [inline common/changes/@uifabric/styling/edwl-2018-10-fixCoachmarkDelay_2018-10-03-21-59.json]: @dzearing is this considered a `patch` or `minor` change?

Comment (KevinTCoughlin) [reply common/changes/@uifabric/styling/edwl-2018-10-fixCoachmarkDelay_2018-10-03-21-59.json]: Typically any `*.types.ts` change is a `minor` bump unless its a breaking API change.

**Pull request merged**